docs/custom/scheduler.mdx

```diff
@@ -3,27 +3,30 @@ id: scheduler
 title: Control threads in AMF
 ---
 
+AMF allows you to schedule and manage the lifespan of threads.
```

Comment:
should this be the the "lifespan of control threads"?

Reply:
nope, the word "control" was used like "manage", AMF allows you to control threads, there is no 'control thread' entity, I've confused you with that one sorry